Tools/gdb/libpython.py

```diff
@@ -634,6 +634,63 @@ def proxyval(self, visited):
         else:
             return BuiltInMethodProxy(ml_name, pyop_m_self)
 
+# Python implementation of location table parsing algorithm
+def read(it):
+    return ord(next(it))
+
+def read_varint(it):
```

Comment:
There is a similar code in test_code, can the repetition be avoided?

Reply:
Unfortunately not.
This code runs in gdb, which is probably running a different version of Python than the one being debugged.